scrapers/Babes.yml

```diff
@@ -0,0 +1,23 @@
+name: "Babes"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.babes.com/scene/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $moreinfo: //div[@class="tjb798-2 flgKJM"]
+    scene:
+      Title: //h1[@class="wxt7nk-4 fSsARZ"]/text()
+      Date:
+        selector: $moreinfo/span[3]/text()
```

Comment:
instead of relying on the order ( span[3] )  we can do it like this
`$moreinfo/span[div[text()='Release Date:']]/text()`

Reply:
Made this update to all of them!